README.md

```diff
@@ -1,2 +1,242 @@
-# learn-user-testing
-Learn how to test your app with real humans 
+# Learn User Experience Testing
+Learn how to test your app with real humans
+
+## Why?
+This is _probably_ the most important/useful skill any "_creative technologist_"
+has. 💪
+
+Well conducted user experience testing helps you ensure that what you are
+planning to build is going to solve the problem for your users that you set out
+to solve. ✅
+
+It prevents you from wasting time ⏳/ effort 😓/ money 💰on guessing 🤔❓what your
+users will want, need or understand by getting them to critique your product as
+you go rather than retrospectively.
+
+Testing usability and following usability guidelines is not something we have to
+do to restrict ourselves or because we're told to. Understanding usability is
+about ensuring users can use our product effectively and so without it our
+product will be less successful. 📉👎  
+
+## What?
+User experience testing is a means of collecting feedback 🗣from users on the
+evolution of your product.
+
+It can be performed multiple times in a cycle of:  🗣❌👎gathering feedback on
+your designs from users -> adapting your designs in light of user feedback 🤔 ✏️
+-> getting feedback on your amended designs to see if they now meet the users'
+needs 🗣✅👍. Once you are satisfied the users needs have been met then you can
+begin to build the agreed designs. 🏗 👷‍♀️ 👷
+
+## How?
+
+There are many approaches to user experience testing that you may choose to use
+based on your product, stage of product development, budget 💵, client base 👥 etc.:
+
+The first stage of user testing you may wish to conduct is known as the discovery
+phase, for information on this stage see: https://www.gov.uk/service-manual/agile-delivery/how-the-discovery-phase-works
+
+To perform usability user testing you may use one/some of the following methods:
+
+* interviews
+* Surveys/ questionnaires
+* Usage pattern analysis using "analytics" data. For more on this see:
+https://github.com/dwyl/learn-analytics 📊
+* Screen recordings #1 🎥🖥
+* App usage/demo video capture
+
+Ensure that when you conduct tests you are testing them across devices so that
+your data reflects all of your users experiences e.g. mobile 📱, tablet and
+desktop 💻 .
+
+### How to conduct interviews
+1. Create a **script** to run through with your testers 📃
```

Comment:
consider adding a sub-bullet point list for the "script" point.
+ As a clear and specific question regarding a goal that a user needs to achieve with the UX/UI being presented.
+ How effectively does the user complete the task without further prompting/help from the UX-tester?
+ Is the flow _immediately intuitive_ or does it require _explanation_?